ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml

```diff
@@ -565,6 +565,7 @@ tests:
       OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      CONFIG_PRIVATE_ENDPOINT_FOR_REGISTRY: "true"
```

Comment:
public registry in a private cluster is still a choice for customers, right? If yes, I think maybe deserve a dedicated installworkflow to cover it.